src/app/modules/status.rs

```diff
@@ -472,7 +472,7 @@ pub(crate) struct StatusModuleVTable {
     /// ### Notes
     /// This is called internally by [`StatusModule::init_settings`]
     #[offset = 0x1A8]
-    pub set_succeeds_bits: extern "C" fn(this: &mut StatusModule, succeeds_bits: u32),
+    pub set_succeeds_bit: extern "C" fn(this: &mut StatusModule, succeeds_bit: i32),
```

Comment:
I'm not sure if this is exported as `StatusModule::set_succeeds_bit`, but if it is make sure to double check the symbol for the arg type if we are changing it. If it's correct that's fine.

Reply:
The symbol for `StatusModule::set_succeeds_bit` is as follows:
```_ZN3app8lua_bind35StatusModule__set_succeeds_bit_implEPNS_26BattleObjectModuleAccessorEi```
The usage of "bit" is singular, so I made sure to reflect that in the function declaration and parameter name. The type for the bit is also signed as indicated by the `i` in the symbol.